chapters/operatorsandexpressions.tex

```diff
@@ -86,6 +86,7 @@ \section{Operator Precedence and Associativity}\label{operator-precedence-and-as
 conditional & \lstinline!if $\mathit{expr}$ then $\mathit{expr}$ else $\mathit{expr}$! & \lstinline!if b then 3 else x!\\ \hline
 named argument & \lstinline!$\mathit{ident}$ = $\mathit{expr}$! & \lstinline!x = 2.26!\\ \hline
 \end{tabular}
+\label{tab:operator-precedence}
 \end{table}
 
 The conditional operator may also include elseif-clauses. Equality \lstinline!=! and
```

Comment:
This comment belongs to the current file, not this particular line.

There is still a `nonnormative` in the start of chapter 3:
> The abbreviated predefined type information below is given as background information for the rest of the presentation.

Reply:
Good point, I tried to figure out what it meant.
It refers to some "predefined type information below", and I cannot figure out what that refers to.
I thus decided to just remove it. It was introduced in Modelica 3.0.

Reply:
Fine.  Resolving.